src/components/MessagesList/MessagesList.vue

```diff
@@ -266,21 +266,16 @@ export default {
 			// Expire older messages when navigating to another conversation
 			this.$store.dispatch('easeMessageList', { token: oldToken })
 			this.stopFetchingOldMessages = false
-			this.messagesGroupedByDateByAuthor = this.prepareMessagesGroups(this.messagesList)
 			this.stickyDate = null
 			this.checkSticky()
```

Comment:
Now we didn't update the messagesList from there, so should also move this?

Reply:
so to check on each messagesList changes ? I don't think it's needed because scroll event is actually triggered when checkSticky needs to be called (e.g: not called on added message when div is not scrollable so there is no sticky date sep but called on added message, if message length changed etc..)

I'd keep it here.